packages/react-menu/etc/react-menu.api.md

```diff
@@ -347,7 +347,7 @@ export function useMenuContextValues(state: MenuState): MenuContextValues;
 export const useMenuDivider: (props: MenuDividerProps, ref: React_2.Ref<HTMLElement>) => MenuDividerState;
 
 // @public (undocumented)
-export const useMenuDividerStyles: (state: MenuDividerState) => ComponentState<MenuDividerSlots>;
+export const useMenuDividerStyles: (state: MenuDividerState) => MenuDividerState;
```

Comment:
With this and the other API file changes, the actual type is unchanged; typescript has just gotten better about avoiding unnecessary type expansion when writing .d.ts files.